exonum/src/blockchain/block.rs

```diff
@@ -138,9 +138,37 @@ pub struct Block {
     pub error_hash: Hash,
     /// Additional information that can be added into the block.
     pub additional_headers: AdditionalHeaders,
+
+    /// No-op field for forward compatibility.
+    #[protobuf_convert(skip)]
```

Comment:
I'd argue that `additional_headers` take care of possible extensions, so this looks redundant.